scss/book-list-detail.scss

```diff
@@ -1,12 +1,49 @@
 .master-container{
   display: flex;
-  max-width: 1366px;
-  width: 100%;
+  height: 768px;
+  width: 1366px;
   background-color: $wild-sand;
-  justify-content: center;
-  align-items: center;
+
+  .navbar-container {
+    position: fixed;
+    display: flex;
+    top: 0;
+    left: 0;
```

Comment:
This class does not have a position specified (so it will be relative by default), so you can remove these rules

Reply:
Those rules need to stay since the position of the navbar is fixed  (line: 8)